crates/scripts-tests/src/tests/utils/layer2.rs

```diff
@@ -0,0 +1,64 @@
+use gw_chain::{
+    chain::{Chain, ProduceBlockParam},
+    mem_pool::{MemPool, PackageParam},
+    next_block_context::NextBlockContext,
+};
+use gw_config::{ChainConfig, GenesisConfig};
+use gw_generator::{
+    account_lock_manage::{always_success::AlwaysSuccess, AccountLockManage},
+    backend_manage::BackendManage,
+    genesis::init_genesis,
+    Generator,
+};
+use gw_store::Store;
+use gw_types::{
+    core::ScriptHashType,
+    packed::{HeaderInfo, Script},
+    prelude::*,
+};
+use parking_lot::Mutex;
+use std::sync::Arc;
+pub const ALWAYS_SUCCESS_ACCOUNT_LOCK_CODE_HASH: [u8; 32] = [42u8; 32];
+
+pub fn setup_chain(rollup_type_script: &gw_types::packed::Script) -> Chain {
```

Comment:
The module name is non-sense, remove the module and move the function to `stake-lock/mod.rs`

Reply:
updated.